README.md

```diff
@@ -45,7 +49,9 @@ MUX_TOKEN_ID=[YOUR_TOKEN_ID]
 MUX_TOKEN_SECRET=[YOUR_TOKEN_SECRET]
 ```
 
-### Add Next Video to `next.config.js`
+### If you choose to do things manually
```

Comment:
I like the idea of kicking this to a different doc we can link to, but we can do that later.